src/main/java/ca/utoronto/fitbook/application/port/out/SaveUserPort.java

```diff
@@ -0,0 +1,8 @@
+package ca.utoronto.fitbook.application.port.out;
+
+import ca.utoronto.fitbook.entity.User;
+
+public interface SaveUserPort
```

Comment:
interface SaveUserPort already exists,

I suggest rebase using "git rebase master" before next commit is pushed